crowdin.yaml

```diff
@@ -1,19 +1,3 @@
-project_identifier: django-girls-tutorial
-api_key: 2be4b1707d0745bc96e9a5c9ffb4ca9b
-base_path: /Users/olasitarska/Desktop/Events/Django Girls/tutorial
-
 files:
-  -
-    source: "/en/**/*.md"
-    translation: "/%two_letters_code%/%original_path%/%original_file_name%"
-    ignore:
```

Comment:
There's no so much sense to ignore files, that are not included in `/en/**/*.md` (and `/_book`, `/node_modules` and `/CONTRIBUTING.md` are not).